installation/sql/postgresql/base.sql

```diff
@@ -343,6 +343,7 @@ INSERT INTO "#__extensions" ("package_id", "name", "type", "element", "folder",
 (0, 'plg_system_sef', 'plugin', 'sef', 'system', 0, 1, 1, 0, 1, '', '', '', 16, 0),
 (0, 'plg_system_sessiongc', 'plugin', 'sessiongc', 'system', 0, 1, 1, 0, 1, '', '', '', 17, 0),
 (0, 'plg_system_skipto', 'plugin', 'skipto', 'system', 0, 1, 1, 0, 1, '', '{}', '', 18, 0),
+(0, 'plg_system_shortcut', 'plugin', 'shortcut', '', 'system', 0, 0, 1, 0, 0, '', '{}', '', 0, 0),
```

Comment:
one extra column. Does empty value is required before system ? 
`(0, 'plg_system_shortcut', 'plugin', 'shortcut', 'system', 0, 0, 1, 0, 0, '', '{}', '', 0, 0),`